addons/website/static/src/components/dialog/add_page_dialog.js

```diff
@@ -359,15 +361,30 @@ export class AddPageTemplates extends Component {
         const activeTabEl = this.tabsRef.el.querySelector(".active");
         const activePaneEl = this.panesRef.el.querySelector(".active");
         activeTabEl?.classList?.remove("active");
+        activeTabEl?.setAttribute("tabIndex", -1);
```

Comment:
Detail but `setAttribute` takes a string parameter, so I would explicitly write `"-1"` and not `-1`. Or use the tabIndex property.